OrbitService/LinuxTracingGrpcHandler.cpp

```diff
@@ -34,6 +38,33 @@ void LinuxTracingGrpcHandler::Start(CaptureOptions capture_options) {
   tracer_->Start();
 
   sender_thread_ = std::thread{[this] { SenderThread(); }};
+
+  if (absl::GetFlag(FLAGS_devmode)) {
+    SetupIntrospection();
+  }
```

Comment:
Probably just a preference, but I would like it better if this was moved to where `LinuxTracingGrpcHandler` is created, and a value that enables this was set from the outside. I don't really like a global flag to modify the behavior of this class without it being visible in its interface.

Reply:
Yeah I have a PR where introspection is set in the capture options so that we can eventually toggle it from the UI. I will add you as a reviewer, thanks!